src/Config.ts

```diff
@@ -99,6 +102,18 @@ export class Config {
         values.Generic.order = Generic.getConfiguration().get<string[]>("order", values.Generic.order);
         values.Generic.customTags = Generic.getConfiguration().get<string[]>("customTags", values.Generic.customTags);
         values.Generic.filteredKeywords = Generic.getConfiguration().get<string[]>("filteredKeywords", values.Generic.filteredKeywords);
+        values.Generic.useGitUserName = Generic.getConfiguration().get<boolean>("useGitUserName", values.Generic.useGitUserName);
+        values.Generic.useGitUserEmail = Generic.getConfiguration().get<boolean>("useGitUserEmail", values.Generic.useGitUserEmail);
+
+        // Check if set to use the git username
+        if (values.Generic.useGitUserName === true) {
+            values.Generic.authorName = gitConfig.UserName.toString();
+        }
+
+        // Check if set to use the git email
+        if (values.Generic.useGitUserEmail === true) {
+            values.Generic.authorEmail = gitConfig.UserEmail.toString();
+        }
```

Comment:
Please move this to `CppDocGen.generateAuthorTag`

Reply:
Moved with https://github.com/cschlosser/doxdocgen/pull/186/commits/0a5167e25756fa381dec2b8aa005077101217580